webextensions/api/devtools.json

```diff
@@ -497,6 +495,81 @@
                   "version_added": false
                 }
               }
+            },
+            "iconPath": {
+              "__compat": {
+                "support": {
+                  "chrome": {
+                    "version_added": true,
+                    "notes": "URL always resolved as absolute."
```

Comment:
The [suggestion in the related PR](https://github.com/mdn/content/pull/20552/files#r1307726220) states this may also be relative to the extension directory and the [extensions docs](https://developer.chrome.com/docs/extensions/reference/devtools_panels/#method-create) say a URL "relative to the extension directory"

Reply:
@bsmth within the context of a web extension they are the same thing e.g. "you can specify the path as an absolute URL, starting at the extension's root, like this: `/path/to/stylesheet.css`" e.g. the extension root is the starting point. Hence we have statements such as "When a content script makes an HTTP(S) request, you _must_ provide absolute URLs." in the web extensions docs.